src/os/bluestore/BlueStore.cc

```diff
@@ -3760,6 +3760,13 @@ int BlueStore::_open_alloc()
   alloc = Allocator::create(cct, cct->_conf->bluestore_allocator,
                             bdev->get_size(),
                             min_alloc_size);
+  if (!alloc) {
+    lderr(cct) << __func__ << "Allocator::unknown alloc type"
+               << cct->_conf->bluestore_allocator
+               << dendl;
+    return -EINVAL;
+  }
```

Comment:
```c++
if (!alloc) {
  lderr(cct) << __func__ << ": unknown bluestore_allocator: " << cct->_conf->bluestore_allocator << dendl;
}
```

could you test this patch, and check the output if the `bluestore_allocator` is not supported. to see why i'd suggest this change?